caffeinated/admin/new/pricing/espresso_events_Pricing_Hooks.class.php

```diff
@@ -1279,7 +1279,7 @@ protected function _get_dtt_attached_tickets_row(
         $datetime,
         $datetime_tickets = array(),
```

Comment:
i was avoiding formatting the files so that there wasn't as much to focus on in the code reviews.
if i autoformat the files, then the LOC (lines of code) changes will go from ~1300 to many many thousands